### ISSUE TITLE
client timeout, server "Error parsing bencode packet, invalid b'bencoded string' type identifier: -62 at position 0"

### ISSUE THREAD

**Issue by rsromanovskiy** [Xpra-org/xpra#4016]
**Describe the bug**
last version of xpra client (install from xpra repository via apt). connection from ubuntu 22.04 nowt work

**To Reproduce**
clientname servername  replaced from real. ssh to server worked.
1.server: `xpra start --exit-with-children --start-child="/tools/Xilinx/Vivado/2021.2/bin/vivado"`
2.client: `xpra attach ssh://servername/0`
3. server: `vim /run/user/1004/xpra/:0.log`
```
2023-09-27 14:44:16,900 cannot access python uinput module:^[[0m
2023-09-27 14:44:16,900  No module named 'uinput'^[[0m
2023-09-27 14:44:16,983 created unix domain socket '/run/user/1004/xpra/servername-0'^[[0m
2023-09-27 14:44:16,983 cannot create group socket '/run/xpra/servername-0'^[[0m
2023-09-27 14:44:16,983  [Errno 13] Permission denied^[[0m
2023-09-27 14:44:16,984 created unix domain socket '/home/username/.xpra/servername-0'^[[0m
2023-09-27 14:44:17,006 pointer device emulation using XTest^[[0m
2023-09-27 14:44:17,039 serving html content from '/usr/share/xpra/www'^[[0m
2023-09-27 14:44:17,294 OpenGL is supported on display ':0'^[[0m
2023-09-27 14:44:17,294  using 'llvmpipe (LLVM 15.0.7, 256 bits)' renderer^[[0m
^[[33m2023-09-27 14:44:17,298 Warning: failed to load the mdns publisher^[[0m
^[[33m2023-09-27 14:44:17,298  No module named 'avahi'^[[0m
^[[33m2023-09-27 14:44:17,298  either install the 'python-avahi' module^[[0m
^[[33m2023-09-27 14:44:17,298  or use the 'mdns=no' option^[[0m
2023-09-27 14:44:17,302 D-Bus notification forwarding is available^[[0m

(Xpra:306696): Gtk-CRITICAL **: 14:44:17.304: gtk_widget_realize: assertion 'widget->priv->anchored || GTK_IS_INVISIBLE (widget)' failed
^[[33m2023-09-27 14:44:17,324 Warning: webcam forwarding is disabled^[[0m
^[[33m2023-09-27 14:44:17,324  the virtual video directory '/sys/devices/virtual/video4linux' was not found^[[0m
^[[33m2023-09-27 14:44:17,324  make sure that the 'v4l2loopback' kernel module is installed and loaded^[[0m
2023-09-27 14:44:17,324 found 0 virtual video devices for webcam forwarding^[[0m
2023-09-27 14:44:17,358 xpra is ready.^[[0m
2023-09-27 14:44:17,358 xpra GTK3 X11 version 3.1 64-bit^[[0m
2023-09-27 14:44:17,384  uid=1004 (username), gid=1004 (username)^[[0m
2023-09-27 14:44:17,384  running with pid 306696 on Linux Ubuntu 22.04 jammy^[[0m
2023-09-27 14:44:17,384  connected to X11 display :0 with 24 bit colors^[[0m
2023-09-27 14:44:19,431 printer forwarding enabled using postscript and pdf^[[0m
2023-09-27 14:44:19,432 started command '/etc/X11/Xsession true' with pid 306805^[[0m
2023-09-27 14:44:19,432 started command '/tools/Xilinx/Vivado/2021.2/bin/vivado' with pid 306806^[[0m
2023-09-27 14:44:19,435 watching for applications menu changes in:^[[0m
2023-09-27 14:44:19,435  '/usr/local/share/applications'^[[0m
2023-09-27 14:44:19,435  '/usr/share/applications'^[[0m
2023-09-27 14:44:19,435  '/var/lib/snapd/desktop/applications'^[[0m
2023-09-27 14:44:19,708 62.6GB of system memory^[[0m

****** Vivado v2021.2.1 (64-bit)
  **** SW Build 3414424 on Sun Dec 19 10:57:14 MST 2021
  **** IP Build 3405791 on Sun Dec 19 15:54:35 MST 2021
    ** Copyright 1986-2021 Xilinx, Inc. All Rights Reserved.

start_gui
2023-09-27 14:44:23,691 Missing property or wrong property type _MOTIF_WM_HINTS (motif-hints)^[[0m
2023-09-27 14:44:23,691  no such window^[[0m
2023-09-27 14:44:25,583 New unix-domain connection received^[[0m
2023-09-27 14:44:25,583  on '/run/user/1004/xpra/servername-0'^[[0m
^[[31m2023-09-27 14:44:25,583 Error parsing bencode packet:^[[0m
^[[31m2023-09-27 14:44:25,583  invalid b'bencoded string' type identifier: -62 at position 0^[[0m
2023-09-27 14:44:25,584 Disconnecting client /run/user/1004/xpra/servername-0:^[[0m
2023-09-27 14:44:25,584  failed to parse bencode packet^[[0m
/usr/lib/python3/dist-packages/gi/overrides/Gtk.py:1689: Warning: g_object_ref: assertion 'G_IS_OBJECT (object)' failed
  return _Gtk_main(*args, **kwargs)
2023-09-27 14:44:26,574 New unix-domain connection received^[[0m
2023-09-27 14:44:26,574  on '/run/user/1004/xpra/servername-0'^[[0m
2023-09-27 14:44:26,574 New unix-domain connection received^[[0m
2023-09-27 14:44:26,574  on '/home/username/.xpra/servername-0'^[[0m
```
**System Information (please complete the following information):**
 - Server OS: "Ubuntu 22.04.3 LTS"
 - Client OS: Ubuntu 22.04.3 LTS
 - Xpra Server Version xpra v3.1
 - Xpra Client Version xpra v5.0.2-r0

**Additional context**
i try another display 0 1 3 999 doesnt metter.
client:
```
 xpra attach ssh://servername/999
2023-09-27 15:16:31,537 Xpra GTK3 X11 client version 5.0.2-r0
2023-09-27 15:16:32,203  running on Linux Ubuntu 22.04 jammy
2023-09-27 15:16:32,204  window manager is 'GNOME Shell'
2023-09-27 15:16:32,854 GStreamer version 1.20.3
2023-09-27 15:16:32,974 created unix domain socket '/run/user/1000/xpra/clients/clientname-41592'
2023-09-27 15:16:33,225 No OpenGL_accelerate module loaded: No module named 'OpenGL_accelerate'
2023-09-27 15:16:33,314 Warning: vendor 'Intel' is greylisted,
2023-09-27 15:16:33,314  you may want to turn off OpenGL if you encounter bugs
2023-09-27 15:16:33,373 OpenGL enabled on 'Mesa Intel(R) Graphics (RPL-P)'
2023-09-27 15:16:33,642 Connected (version 2.0, client OpenSSH_8.9p1)
2023-09-27 15:16:34,108 Authentication (publickey) successful!
2023-09-27 15:16:35,665 ssh server OS is 'linux-gnu'
2023-09-27 15:16:37,408 paramiko SSH agent forwarding enabled
2023-09-27 15:16:37,446  keyboard settings: rules=evdev, model=pc105, layout=us,ru,us
2023-09-27 15:16:37,454  desktop size is 4000x2560:
2023-09-27 15:16:37,455   :1.0 (1058x677 mm - DPI: 96x96) workarea: 3930x1988 at   70x572 
2023-09-27 15:16:37,455     DEL DP-3         1440x2560 at 2560x0    (597x336 mm - DPI: 61x194) workarea: 1440x2560 at 2560x0
2023-09-27 15:16:37,455     DEL DP-1         2560x1440 at    0x545  (597x336 mm - DPI: 109x109) workarea: 2490x1413 at   70x572
2023-09-27 15:17:07,438 connection timed out
2023-09-27 15:17:07,452 SSH EOF on stderr of run-xpra
2023-09-27 15:17:07,460 removing unix domain socket '/run/user/1000/xpra/clients/clientname-41592'
```

**Comment by rsromanovskiy** [comment]
reinstall to 3.1 client fix problem, but at start version 3.1 cant connect with another error something about "xpra initialization error: connection failed: 'NoneType' object has no attribute '_fields"

**Comment by totaam** [comment]
> `xpra GTK3 X11 version 3.1 64-bit`

This is not a supported version.
Versions older than 5.x are EOL: https://github.com/Xpra-org/xpra/wiki/Versions
